_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+The Townships are made up of six square miles, with each square mile known as a Section. These Sections are eventually subdivided down to an acre by way of Half Sections, Quarter Sections, Quarter-Quarter Sections, etc. (Fig 6). An acre, again, is simply 10 square chains - and there are 640 acres in a Section (a square-mile) or 40 acres in a quarter-quarter section. Hence the phrase, “40 acres and mule” is all you needed to be self-sufficient.
+
+<figure class="caption caption--right">
+  <img src="{{ "/images/townships_sections.png" | prepend: site.baseurl }}" alt="Townships and Sections" />
+  <figcaption class="caption__text">Fig 6: Townships, Ranges and Sections</figcaption>
+</figure>
+
+No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66 foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy - and many early surveyors had a shaky grasp of mathematics and they were required to work quickly.
+
+These early surveyors typically placed a permanent monument at Section and Quarter-Section corners. These markers are still heavily used today and are the starting point for almost every Western legal description you’ll encounter - including land deeds (the transfer of private property), water rights (well locations), taxing areas, municipal annexations - and consequently voting districts. Because of this, it has been difficult for the US to adopt the International System of Units (the modern metric system).
+
+It is precisely this survey pattern that explains why the Western United States is so orthogonal. Although the grid is not unique to the West, it's certainly at the core of all land sales and as a result it influences many development patterns. Let’s look at a few examples.
+
+## The Western Grid
+{: .text-left}
+
+<figure class="caption caption--left caption--flow">
+  <a href="{{ "/images/plat_of_zion.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/plat_of_zion_sm.png" | prepend: site.baseurl }}" class="width: 100%;" alt="Plat of Zion">
+  </a>
+  <figcaption class="caption__text">Fig 7: Plat of Zion</figcaption>
+</figure>
+<figure class="caption caption--right caption--flow">
+  <a href="{{ "/images/ColonialBoston.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/ColonialBoston_sm.png" | prepend: site.baseurl }}" alt="Colonial Boston">
+  </a>
+  <figcaption class="caption__text">Fig 8: Colonial Boston</figcaption>
```

Comment:
Footnote 8: 
John Bonner, ca. 1643-1726; William Price, fl. 1725-1769; and Francis Dewing, fl. 1716-1722, “The town of Boston in New England,” Map, 1723, Norman B. Leventhal Map & Education Center, accessed March 01, 2019. https://collections.leventhalmap.org/search/commonwealth:9s161f21f.